src/read/config.rs

```diff
@@ -0,0 +1,23 @@
+/// Configuration for reading ZIP archives.
+#[derive(Debug, Default, Clone)]
```

Comment:
For now, make this `#[repr(transparent)]`, and derive `Copy` (we can always remove it later if we add non-`Copy` fields).

Reply:
I was hesitant to add `Copy` as it's a breaking change to remove it in the future. The compiler ought to be smart about it. But I'll add it.